storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteObjects.java

```diff
@@ -22,14 +23,14 @@
 public class PreparedDeleteObjects<T> extends PreparedDelete<DeleteResults<T>> {
 
     @NonNull
-    private final Collection<T> objects;
+    private final Iterable<T> objects;
```

Comment:
It's more general and in other places we accept Iterable

вс, 12 апр. 2015, 0:29, Dmitrii Nikitin <notifications@github.com>:

> In
> storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/delete/PreparedDeleteObjects.java
> <https://github.com/pushtorefresh/storio/pull/275#discussion_r28198567>:
>
> > @@ -22,14 +23,14 @@
> >  public class PreparedDeleteObjects<T> extends PreparedDelete<DeleteResults<T>> {
> >
> >      @NonNull
> > -    private final Collection<T> objects;
> > +    private final Iterable<T> objects;
>
> Why have you decided to use iterable instead of collection?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pushtorefresh/storio/pull/275/files#r28198567>.
>